docs/DeveloperGuide.md

```diff
@@ -154,9 +154,9 @@ The sections below give more details of each component.
 
 The **API** of this component is specified in [`Ui.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/ui/Ui.java)
 
-![Structure of the UI Component](images/UiClassDiagram2.png)
+![Structure of the UI Component](images/UiClassDiagram.png)
 
-The UI consists of a `MainWindow` that is made up of parts e.g.`CommandBox`, `ResultDisplay`, `ResidentTableView`, `StatusBarFooter` etc. All these, including the `MainWindow`, inherit from the abstract `UiPart` class which captures the commonalities between classes that represent parts of the visible GUI.
+The UI consists of a `MainWindow` that is made up of parts e.g.`CommandBox`, `ResultDisplay`, `ResidentTab`, `BookingTab`, `ResidentTableView`, `BookingTableView`, `StatusBarFooter` etc. All these, including the `MainWindow`, inherit from the abstract `UiPart` class which captures the commonalities between classes that represent parts of the visible GUI.
```

Comment:
We don't actually have `ResidentTab`, `BookingTab` and `StatusBarFooter` classes.

We do however have `Tab` objects with names `residentTab` and `venueTab` in an `MainWindow` object.

Reply:
It might be best to not include classes like `ResidentTableView` and `BookingTableView` which are part of each tab.

What I mean by this is just to mention that `MainWindow` is made up of parts, e.g. `CommandBox`, `ResultDisplay`, `ResidentTabView`, `VenueTabView`, `CurrentWorkingFileFooter` and `HelpWindow`. Then you can mention that `ResidentTabView` is further made up of `ResidentTableView` and `VenueTabView` is further made up of `VenueListView` and `BookingTableView`. Though this might be redundant with the diagram.